hack/machineset.sh

```diff
@@ -173,6 +173,56 @@ $(get_spec $infraID $az $provider)
 EOF
 }
 
+# get_gcp_ms creates a MachineSet for Google Cloud
+get_gcp_ms() {
+  if [ "$#" -lt 4 ]; then
+    error-exit incorrect parameter count for get_gcp_ms $#
+  fi
+
+  local infraID=$1
+  local region=$2
+  local az=$3
+  # For GCP the zone field returns the region + zone, like: `us-central1-a`.
+  # Installer created Machinesets only append the `-a` portion, so we should do the same.
+  local az_suffix=$(echo $az |awk -F "-" '{print $NF}')
+  local provider=$4
+  local projectID=$(cat ~/.gcp/osServiceAccount.json | jq -r '.project_id')
+  local imageName=$(gcloud compute images list --filter="family = windows-2022-core" --sort-by="~creationTimestamp" --standard-images --limit=1 | tail -n1 |awk '{print $1}')
```

Comment:
You can just do `gcloud compute images list --filter="family = windows-2022-core" --sort-by="~creationTimestamp" --standard-images --limit=1 --format="value(NAME)"`